app/models/miq_provision_virt_workflow.rb

```diff
@@ -198,10 +198,12 @@ def validate_sysprep_field(field, values, dlg, fld, value)
   end
 
   def default_require_sysprep_enabled(_field, _values, dlg, fld, value)
+    # Used by user. Do not erase.
```

Comment:
We should make these comments more descriptive: `# This method is available as a dialog field validation.  Do not erase.`